inc/formanswer.class.php

```diff
@@ -862,13 +863,16 @@ public function generateTarget(): bool {
 
             // Generate the target
             $generatedTarget = $targetObject->save($this);
-            if ($generatedTarget === null) {
-               $success = false;
-               break;
+            if ($targetObject->isObjectTarget()) {
+               if ($generatedTarget === null) {
+                  // If the target generates an object but none generated
+                  $success = false;
+               } else {
+                  $this->targetList[] = $generatedTarget;
+                  // Map [itemtype of the target] [item ID of the target] = ID of the generated target
+                  $generatedTargets->addTarget($targetObject, $generatedTarget);
```

Comment:
populate targetList only for generataed objects, and feed the composite tickets linker only in this case.